docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-kcp.md

```diff
@@ -32,21 +43,45 @@ The kubectl plugin lines use fully specific executables (e.g.,
 `kubectl kubestellar prep-for-syncer` corresponds to
 `bin/kubectl-kubestellar-prep_for_syncer`).
 
+### Initialize the KubeStellar platform
+
+In this step KubeStellar creates and populates the Edge Service
+Provider Workspace (ESPW), which exports the KubeStellar API, and also
+augments the `root:compute` workspace from kcp TMC as needed here.
+Those augmentation consists of adding authorization to update the
+relevant `/status` and `/scale` subresources (missing in kcp TMC) and
+extending the supported subset of the Kubernetes API for managing
+containerized workloads from the four resources built into kcp TMC
+(`Deployment`, `Pod`, `Service`, and `Ingress`) to the other ones that
+are namespaced and are meaningful in KubeStellar.
+
+```shell
+kubestellar init
+```
+
+### Create an inventory management workspace.
+```shell
+kubectl ws root
+kubectl ws create imw-1 --enter
```

Comment:
The `--enter` is not needed, since the next command sets the currrent workspace.

Reply:
was simply copied from previous md